students/WesleyM/Lesson4/trigrams.py

```diff
@@ -0,0 +1,42 @@
+import re
+import random
+
+def open_file(string):
+    '''Opens, reads and closes a .txt file'''
+    f = open(string)
+    secret_data = f.read()
+    f.close()
+    return secret_data
+
+def ngrams(input, n):
```

Comment:
Cool idea to make it more flexible like this so you can experiment with different word group lengths and see what kind of results you get